.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Unit Tests
+
+on:
+  push:
+    branches:
+    - "main"
+  pull_request:
+    branches:
+    - "main"
+
+jobs:
+  unit-tests:
+    name: Unit Tests
```

Comment:
This is going to come out looking like `Unit Tests / Unit Tests` by the way, not sure if that's desirable. `Unit Tests / Ginkgo`, `Unit Tests / runner units`, `CI / runner unit test` might be preferable...

Reply:
Updated to be less repetative.